inc/saiport.h

```diff
@@ -1713,6 +1796,74 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_AUTO_NEG_FEC_MODE_OVERRIDE,
 
+    /**
+     * @brief MDIX mode status for the port
+     *
+     * @type sai_port_mdix_mode_status_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_MDIX_MODE_STATUS,
+
+    /**
+     * @brief MDIX mode configuration for the port
+     *
+     * @type sai_port_mdix_mode_config_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MDIX_MODE_CONFIG_AUTO
+     */
+    SAI_PORT_ATTR_MDIX_MODE_CONFIG,
+
+    /**
+     * @brief Configure auto negotiation configuration mode for the port
+     *
+     * @type sai_port_auto_neg_config_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MDIX_MODE_CONFIG_AUTO
+     * @condition SAI_PORT_ATTR_AUTO_NEG_MODE == true
+     */
+    SAI_PORT_ATTR_AUTO_NEG_CONFIG_MODE,
+
+    /**
+     * @brief Enable auto detection between 1000X and SGMII slave mode
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @condition SAI_PORT_ATTR_SPEED == 1000 or SAI_PORT_ATTR_SPEED == 100 or SAI_PORT_ATTR_MEDIA_TYPE == SAI_PORT_ATTR_MEDIA_TYPE_FIBER
+     */
+    SAI_PORT_ATTR_1000X_SGMII_SLAVE_AUTODETECT,
+
+    /**
+     * @brief Configure EEE latency mode
+     *
+     * False: Fixed latency
+     * True: Variable latency
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_EEE_LATENCY_MODE,
+
+    /**
+     * @brief Configure Fiber module type
+     *
+     * @type sai_port_module_type_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MODULE_TYPE_1000BASE_X
+     * @condition SAI_PORT_ATTR_SPEED == 1000 or SAI_PORT_ATTR_SPEED == 100 or SAI_PORT_ATTR_MEDIA_TYPE == SAI_PORT_ATTR_MEDIA_TYPE_FIBER
```

Comment:
value for SAI_PORT_ATTR_MEDIA_TYPE  must be one of SAI_PORT_MEDIA_TYPE_*

Reply:
also condition attribute must be mandatory on create, or you can change this to validonly

Reply:
So there is no way to have conditions with multiple attributes. As discussed last week, we would like to allow this attribute when media type is set to fiber and speed is either 1000 or 100. How do I do that?